package.json

```diff
@@ -26,10 +27,12 @@
   "author": "Ben Coe <ben@npmjs.com>",
   "license": "ISC",
   "dependencies": {
+    "@types/istanbul-lib-coverage": "^2.0.1",
     "convert-source-map": "^1.6.0",
     "source-map": "^0.7.3"
   },
   "devDependencies": {
+    "@types/node": "^12.0.10",
```

Comment:
this is a dev dep since it pollutes the global namespace (https://github.com/facebook/jest/issues/8092)